src/ClaudeAPI.hs

```diff
@@ -1,9 +1,11 @@
-module ClaudeAPI 
-( -- Configurations
+{-# LANGUAGE DuplicateRecordFields #-}
+
+module ClaudeAPI
+( -- ** Configurations
```

Comment:
The *s promote these comments ignored by compiler into haddock comments, that define titles in the generated documentation (see haddock [markup](https://haskell-haddock.readthedocs.io/latest/markup.html#documentation-structure-examples) doc)